samples/system-test/quickstart.test.js

```diff
@@ -45,4 +47,15 @@ describe('Quickstart sample tests', () => {
     assert.ok(exists);
     await file.delete();
   });
+
+  it('should get assets history successfully', async () => {
+    const assetName = util.format('//storage.googleapis.com/%s', bucketName);
```

Comment:
You don't need to use `util.format` here :) A simple `//storage.googleapis.com/${bucketName}` will work fine.  NOTE: wrap that thing in backticks.  Markdown formatting won't let me 😆